### PR TITLE
Refacto module controller to extend PrestaShopAdminController

### LINKED ISSUE #36843: Refacto module controllers
- ModuleController
- UpdatesController
- AlertsController

### PR DESCRIPTION
**Pull request by jolelievre**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Refacto module controllers
| Type?             | refacto
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI green and UI tests green
| UI Tests          | https://github.com/jolelievre/ga.tests.ui.pr/actions/runs/10771373283
| Fixed issue or discussion?     | Fixes #36843
| Related PRs       | ~
| Sponsor company   | ~

### CODE REVIEW

**Review by matthieu-rolland — APPROVED** [review]

**Review by boherm — APPROVED** [review]

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
